src/app/templates/src/_plugin.js

```diff
@@ -3,6 +3,9 @@ import videojs from 'video.js';
 // Default options for the plugin.
 const defaults = {};
 
+// Cross-compatibility for Video.js 5 and 6.
+const registerPlugin = videojs.registerPlugin || videojs.plugin;
```

Comment:
Maybe we should include a commented out videojs.dom here too? Just for reference?